src/qt/pivx/settings/forms/settingsfaqwidget.ui

```diff
@@ -1239,7 +1239,7 @@
                            &lt;li&gt;
                            Once all those steps are followed staking should be enabled.
                            &lt;li&gt;
-                           You can see the status of staking in the wallet by mousing over the package icon in the row on the top left of the wallet interface. There package will be lit up and will state &quot;Staking Enabled&quot; to indicate it is staking.  Using the command line interface (pivx-cli); the command &lt;i&gt;getstakingstatus&lt;/i&gt; will confirm that staking is active.
+                           You can see the status of staking in the wallet by mousing over the package icon in the row on the top left of the wallet interface. There package will be lit up and will state &quot;Staking Enabled&quot; to indicate it is staking.  Using the command line interface (zenzo-cli); the command &lt;i&gt;getstakingstatus&lt;/i&gt; will confirm that staking is active.
```

Comment:
"There package will be lit up"? 

Grammar Suggestions: There, the package will be lit up

Proper Suggestion: There, replace package icon with staking icon and for the second sentence... 

There, the staking icon will be glowing white and will state &quot;Staking Enabled&quot; to indicate it is staking.

Reply:
Could be amended in a seperate PR.